autogen/code_utils.py

```diff
@@ -223,6 +223,31 @@ def _cmd(lang):
     raise NotImplementedError(f"{lang} not recognized in code execution")
 
 
+class CommandSanitizer:
```

Comment:
Suggest moving this to `coding` module. Instead of `code_utils.py`, which will be deprecated in the future after switch to new code executors is completed. 

Also, I think we can start out with simple utility method rather than a class to make things simpler.

Reply:
Good to know - i've made it a function with `LocalCommandlineCodeExecutor`